deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -522,6 +522,14 @@ spec:
           value: "3600"
         - name: VERIFY_SSL
           value: "true"
+        - name: EXCLUDE_NAMESPACE_REGEX
+          value: ""
+        - name: EXCLUDE_CONTAINER_REGEX
+          value: ""
+        - name: EXCLUDE_HOST_REGEX
+          value: ""
+        - name: EXCLUDE_POD_REGEX
+          value: ""
```

Comment:
Hmm, this is from https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/206, I'm surprised the CI didn't trigger this in that PR, might be a bug. We'll have to take a look.